code/game/machinery/camera/camera.dm

```diff
@@ -129,6 +129,18 @@
 	if(!health && status)
 		deactivate(user, 0)
 
+/obj/machinery/camera/attack_animal(mob/living/simple_animal/user)
+	if(!istype(user))
+		return
+	if(user.melee_damage_upper >= 10)
```

Comment:
Why 10? I mean that's a lot as far as animals go. None of the pets have above 9. Also, you saying poly can't cut a camera?